pyteal/ast/acct_test.py

```diff
@@ -1,5 +1,3 @@
-import pytest
-
 from .. import *
 
 # this is not necessary but mypy complains if it's not included
```

Comment:
Since you're already touching a lot of files, would you mind looking into whether mypy still needs these superfluous imports? I suspect it doesn't, since these are relics from an older version of mypy, and that we'd be able to remove them.

Reply:
@jasonpaulos @tzaffi I attempted to address the feedback via https://github.com/algorand/pyteal/pull/277.  Can you have a look?

Reply:
I've merged this in. Thank you!